components/support/locale/src/test/java/mozilla/components/support/locale/LocaleUseCasesTest.kt

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.locale
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.browser.state.action.UpdateLocaleAction
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.support.test.mock
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import java.util.Locale
+
+@RunWith(AndroidJUnit4::class)
+class LocaleUseCasesTest {
+
+    private lateinit var browserStore: BrowserStore
+
+    @Before
+    fun setup() {
+        browserStore = mock()
+    }
+
+    @Test
+    fun `UpdateLocaleUseCase`() {
```

Comment:
nit: The test names should describe the behavior not the methods that are called. If we change the method names later, all the tests will have wrong names :). 

Better to use the naming convention `WHEN, THEN`,  you could find an example [here](https://github.com/mozilla-mobile/android-components/blob/d017f023c9c622f7c590129df4b78580c5f9e325/components/browser/menu/src/test/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilderTest.kt#L34).

Maybe `WHEN notifying the locale has changed THEN the locale state should be updated`.